server/store/datastore/build_test.go

```diff
@@ -287,34 +288,20 @@ func TestBuilds(t *testing.T) {
 }
 
 func TestBuildIncrement(t *testing.T) {
-	store, closer := newTestStore(t, new(model.Build), new(model.Repo))
+	store, closer := newTestStore(t, new(model.Build))
 	defer closer()
 
-	repo := &model.Repo{
-		UserID:   1,
-		FullName: "bradrydzewski/test",
-		Owner:    "bradrydzewski",
-		Name:     "test",
-	}
-	if err := store.CreateRepo(repo); err != nil {
-		t.Error(err)
+	buildA := &model.Build{RepoID: 1}
+	if !assert.NoError(t, store.CreateBuild(buildA)) {
```

Comment:
What is the purpose of this if?

Reply:
on first error of test fail hard -> else it could panic